code-challenges.js

```diff
@@ -7,43 +7,112 @@
 
-const temp1 = 42
-const temp2 = 350
-const temp3 = 212
+//This function takes any given number and determines if it is below, at, or above boiling point by using greater than/less than/equal to comparisons.
+//The boiling point is 212 degrees Fahrenheit. 
+
+//The input will be temperatures assigned to variables labeled temp1, temp2, and temp3.
+ const temp1 = 42
+ const temp2 = 350
+ const temp3 = 212
+
+const givenNumber = (temperature) => {
+    if (temperature < 212) {
+         return `${temperature} is below boiling point`
+     } else if (temperature === 212) {
+        return `${temperature} is at boiling point`
+    } else if (temperature > 212) {
+        return `${temperature} is above boiling point`
+   }
+ }
+//Console.log will be used to display the return values of the function when executed. 
+ console.log(givenNumber(temp1))
+ console.log(givenNumber(temp2))
+ console.log(givenNumber(temp3))
+
+//The expected output is a given number and its relation to boiling point (i.e.,number is: at boiling point, below boiling point, above boiling point):
+//42 is below boiling point
+//350 is above boiling point
+//212 is at boiling point
 
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
+//This code combines two assigned arrays and returns the total length. 
+//The arrays will be named myNumbers1 and myNumbers2, and will contain the same amount of numbers. 
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+//Console.log will be used to display the output with concatenation and length methods implemented. 
+console.log(myNumbers1.concat(myNumbers2).length)
+//The expected output is: 10 
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
+// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 ovarB"
+
+//This code reverses the letters of the given string, "Bravo 2022". The string will be contained in the variable called currentCohort. 
 const currentCohort = "Bravo 2022"
+//Console.log will be used to display the output with split and join methods implemented. 
+console.log(currentCohort.split("").reverse().join(""))
+//The expected output is: 2022 ovarB
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
-const myArray = [13, 34, 5, 10, 27, 42]
+//This JavaScript statement evaluates whether each number in a given array is even or odd. 
+//The array will be named myArray. 
+  const myArray = [13, 34, 5, 10, 27, 42]
+//An if/else statement will be used to determine if the numbers are even or odd. 
+ for(let i = 0; i < myArray.length; i ++){
```

Comment:
Make sure to add iteration into your pseudocode.  Pseudocode should break down each step in the process and the different methods or process needed to achieve the result.